src/libcore/num/mod.rs

```diff
@@ -329,6 +329,27 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Returns the number of leading ones in the binary representation of `self`.
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "let n = -1", stringify!($SelfT), ";
```

Comment:
I think it'd be cleaner to write this in binary notation so the reader can just count the digits (instead of assuming they know that Rust requires two's complement integers). This of course means you can you 8 digits at most so it works in this macro :)

Reply:
@killercup good one; done!